GitUI/GitUICommands.cs

```diff
@@ -1471,6 +1471,8 @@ private bool RunCommandBasedOnArgument(IReadOnlyList<string> args, IReadOnlyDict
                     return StartViewPatchDialog(args.Count == 3 ? args[2] : "");
                 case "uninstall":
                     return UninstallEditor();
+                case ".":
```

Comment:
see comment on line 1383: Update FormCommandlineHelp too

Reply:
Yes, I have added that. Thanks for the knowledge share.